coordinate.go

```diff
@@ -376,7 +377,7 @@ func (a *Agent) updateNodeCoordinate(node *api.Node, rtt time.Duration) error {
 	if err != nil {
 		return fmt.Errorf("error applying coordinate update for node %q: %v", node.Node, err)
 	}
-	a.logger.Printf("[INFO] Updated coordinates for node %q with distance %q from previous", node.Node, coord.Coord.DistanceTo(newCoord))
+	a.logger.Info("Updated coordinates for ", "node", hclog.Fmt("%q with distance %q from previous", node.Node, coord.Coord.DistanceTo(newCoord)))
```

Comment:
Suggestion to split coord.Coord.DistanceTo(newCoord) into it's own k/v pair with a key something like 'distanceFromPreviousCoord'